Src/Support/Google.Apis.Auth/OAuth2/Responses/TokenResponse.cs

```diff
@@ -93,6 +93,21 @@ public DateTime Issued
         [Newtonsoft.Json.JsonPropertyAttribute(Order = 2)]
         public DateTime IssuedUtc { get; set; }
 
+        /// <summary>Access token for impersonated credentials.</summary>
+        [Newtonsoft.Json.JsonPropertyAttribute("accessToken")]
+        private string ImpersonatedAccessToken { set { AccessToken = value; } }
```

Comment:
Expression-bodied members make this slight easier:

```csharp
private string ImpersonatedAccessToken { set => AccessToken = value; }
```

It's odd to have a setter-only property though. What's the reasoning for that?

Reply:
The token response from IAM endpoint uses different JSON key. 

Access token response: `{ "accessToken": "eyJ0eXAi...NiJ9", "expireTime": "2020-04-07T15:01:23.045123456Z"}`
while TokenResponse expects: `{ "access_token": "...", "expires_in": 3600}"`

Id token response: `{ "token": "eyJ0eXAi...NiJ9" }`
while TokenResponse expects: `{"id_token": "..." }`

So in this PR, when we parse the JSON, we assign the `accessToken` JSON value to `access_token`, and `token` JSON value to `id_token`. After that these non-standard JSON keys are no longer used, so we don't need a getter.

Also we need to convert `"2020-04-07T15:01:23.045123456Z"` expireTime to `expires_in`.